chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts

```diff
@@ -7,17 +7,20 @@ export class MockCustomerService {
 
   constructor() { }
   public generateCustomer() {
-    // TODO: create a new customer.
-    const newCust = {
-      firstName: this.generateFirstName(),
-      lastName: this.generateLastName(),
-      email: this.generateEmail(),
+    let generateCustomer;
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
+    const newCustomer = {
+      firstName,
+      lastName,
+      email: this.generateEmail(firstName, lastName),
       phone: this.generatePhone(),
       city: this.generateCity(),
       state: this.generateState(),
       id: this.generateId()
     };
-    return newCust;
+    generateCustomer = newCustomer;
```

Comment:
No need for this.  Just return `newCustomer`.

Reply:
Removed genCustomer.